### PR TITLE
docs: Retire Toucan. Generate CSV version of schema locally.

### PR DESCRIPTION
**Pull request by jpmckinney**
I've tested locally, and the `release-schema.csv` file is correctly generated and translated.

### CODE REVIEW

**Comment by jpmckinney** [comment]
Once merged, I'll deploy to the live server.

File: docs/locale/es/LC_MESSAGES/schema/reference.po [hunk 1]
```diff
@@ -72,9 +72,7 @@ msgid ""
 "This page presents the release schema in tables, with additional information"
 " in paragraphs. You can also download the canonical version of the release "
 "schema as [JSON Schema](../../build/current_lang/release-"
-"schema.json), download it as a [CSV spreadsheet](https://toucan.open-"
-"contracting.org/mapping-sheet/?source=https://standard.open-"
-"contracting.org/1.1/en/release-schema.json), view it in an [interactive "
+"schema.json), download it as a [CSV spreadsheet](../../build/current_lang/release-schema.json), view it in an [interactive "
```
Comment (yolile) [inline docs/locale/es/LC_MESSAGES/schema/reference.po]:
```suggestion
"schema.json), download it as a [CSV spreadsheet](../../build/current_lang/release-schema.csv), view it in an [interactive "
```
? 
Same for the other languages

Comment (jpmckinney) [reply docs/locale/es/LC_MESSAGES/schema/reference.po]: Oops, fixed now.

**Review by yolile — APPROVED** [review]
It works for me locally too

**Pull request merged**